coreax/coresubset.py

```diff
@@ -274,7 +273,8 @@ def _greedy_body(
             iteration ``i``.
         :param x: :math:`n \times d` data matrix
         :param kernel_vectorised: Vectorised kernel computation function. This should be
-            the :meth:`compute` method of a :class:`~coreax.kernel.Kernel` object,
+            the :meth:`~coreax.kernel.Kernel.compute` method of a
+            :class:`~coreax.kernel.Kernel` object,
```

Comment:
I thought the linter enforced no punctuation at the end of parameter descriptions, but maybe not. In any case, to be consistent, the trailing comma probably wants removing here.